content/guides/initiating-refunds.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Initiating Refunds
+img: /background.png
```

Comment:
This is not a default. We are specifying which image to render here.

Reply:
I have updated your suggestions. Thanks